inference-engine/tests/functional/shared_test_classes/src/subgraph/concat_quantization_during_memory_requantization.cpp

```diff
@@ -99,6 +99,16 @@ namespace SubgraphTestsDefinitions {
         function = std::make_shared<ngraph::Function>(sigm, input, "concat_quant_during_memory_requant_nomemory");
     }
 
+    void ConcatQuantDuringMemoryRequantTest::LoadNetwork() {
+        LayerTestsUtils::LayerTestsCommon::LoadNetwork();
+        inferRequest = executableNetwork.CreateInferRequest();
```

Comment:
I suppose this `inferRequest` is on purpose created here in advance.
Is it ok that it will be overwritten in `Infer`? (interferes with the comment above)

Reply:
It shouldn't be overwritten by Infer, I override Infer method (line 108) to avoid it.

Reply:
Upd.: just noted they go as pair. This must be inevitable due to the nature of the test, I guess?

Reply:
yes

Reply:
This design looks a bit fragile. Can some sort of rearrangement be thought of (so that `Infer` wouldn't have to be overridden; out of this PR of course)?